jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -163,19 +172,17 @@ public void close() {
 
     private String operationName = null;
     private long startTimeMicroseconds;
-    private SpanContext parent;
+    private List<Reference> references = new ArrayList<Reference>();
     private final Map<String, Object> tags = new HashMap<String, Object>();
+    private final Map<String, String> baggage = new HashMap<String, String>();
```

Comment:
PS - if you prefer, we can address these in the follow-up PRs, since this one is pretty large.